Source/Core/BoxGeometry.js

```diff
@@ -47,29 +51,42 @@ define([
      * @example
      * var box = new Cesium.BoxGeometry({
      *   vertexFormat : Cesium.VertexFormat.POSITION_ONLY,
-     *   maximumCorner : new Cesium.Cartesian3(250000.0, 250000.0, 250000.0),
-     *   minimumCorner : new Cesium.Cartesian3(-250000.0, -250000.0, -250000.0)
+     *   maximum : new Cesium.Cartesian3(250000.0, 250000.0, 250000.0),
+     *   minimum : new Cesium.Cartesian3(-250000.0, -250000.0, -250000.0)
      * });
      * var geometry = Cesium.BoxGeometry.createGeometry(box);
      */
     var BoxGeometry = function(options) {
         options = defaultValue(options, defaultValue.EMPTY_OBJECT);
-        var min = options.minimumCorner;
-        var max = options.maximumCorner;
+
+        var min = options.minimum;
+        var max = options.maximum;
 
         //>>includeStart('debug', pragmas.debug);
         if (!defined(min)) {
-            throw new DeveloperError('options.minimumCorner is required.');
+            if (defined(options.minimumCorner)) {
+                min = options.minimumCorner;
+                deprecationWarning('boxGeometry', 'options.minimumCorner is deprecated. Use options.minimum instead');
```

Comment:
Let's stick with consistent naming.  Classes use Pascal class so this should be `BoxGeometry`.  Functions and variables use `camelCase`.

Same comment for `BoxOutlineGeometry` below.